keras/callbacks.py

```diff
@@ -2377,6 +2378,48 @@ class TensorBoard(Callback, version_utils.TensorBoardVersionSelector):
     # Then run the tensorboard command to view the visualizations.
     ```
 
+    Custom batch-level summaries in a subclassed Model:
+
+    ```python
+    class MyModel(tf.keras.Model):
+
+      def build(self, _):
+        self.dense = tf.keras.layers.Dense(10)
+
+      def call(self, x):
+        outputs = self.dense(x)
+        tf.summary.histogram('outputs', outputs)
+        return outputs
+
+    model = MyModel()
+    model.compile('sgd', 'mse')
+
+    # Make sure to set `update_freq=N` to log a batch-level summary every N
+    # batches.  In addition to any `tf.summary` contained in `Model.call`,
+    # metrics added in `Model.compile` will be logged every N batches.
```

Comment:
Is it N batches or 1?

Reply:
(same below)

Reply:
N is correct, I'll add a note to `update_freq` documentation about `Model.compile`